### PR TITLE
:lipstick: feat: Create reusable content text component #27

### PR DESCRIPTION
**Pull request by thadeusdev**
# Description

Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

Reference the issue related to this PR as shown below. Every issue has it's own number you can find the issue numbers [here](https://github.com/italanta/italanta-apps/issues).

Fixes # (issue) - *Example*: Fixes #10, #20 NOTE: GitHub issue numbers should be prefixed with **#** to aid with issue tracking.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**: (optional)

- Firmware version:
- Hardware:
- Toolchain:
- SDK:

## Checklist

- [✓ ] My code follows the style guidelines of this project
- [✓ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

## URL for Create reusable content text component #27
[local host](http://localhost:4200/content-text).

screen-view
tablet-view
phone-view

### CODE REVIEW

File: libs/data/ui/content-text/src/lib/mock-content-text.data.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import { ContentText } from '@elewa-website/models/schema/ui/texts';
```
Comment (LemmyMwaura) [inline libs/data/ui/content-text/src/lib/mock-content-text.data.ts:1]: Good👍

File: libs/elements/layout/texts/src/lib/components/elewa-text-content-item/elewa-text-content-item.component.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  selector: 'elewa-website-elewa-text-content-item',
+  templateUrl: './elewa-text-content-item.component.html',
+  styleUrls: ['./elewa-text-content-item.component.scss'],
+})
+export class ElewaTextContentItemComponent implements OnInit {
+  @Input() content!: ContentText[];
+
+  ngOnInit(){
+    this.content = mockContent;
+  }
```
Comment (LemmyMwaura) [inline libs/elements/layout/texts/src/lib/components/elewa-text-content-item/elewa-text-content-item.component.ts:16]: Good 👍

File: libs/models/schema/ui/texts/src/lib/content-text.interface.ts [hunk 3]
```diff
@@ -0,0 +1,4 @@
+export interface ContentText {
+  title: string;
+  descriptions: string[];
+}
```
Comment (LemmyMwaura) [inline libs/models/schema/ui/texts/src/lib/content-text.interface.ts:4]: 👍

**Review by LemmyMwaura — APPROVED** [review]
Very Good Work. Just update your Pr description next time.

File: libs/data/ui/content-text/src/lib/mock-content-text.data.ts (hunk 1 above)
Comment (thadeusdev) [reply libs/data/ui/content-text/src/lib/mock-content-text.data.ts:1]: Thank you

File: libs/elements/layout/texts/src/lib/components/elewa-text-content-item/elewa-text-content-item.component.ts (hunk 2 above)
Comment (thadeusdev) [reply libs/elements/layout/texts/src/lib/components/elewa-text-content-item/elewa-text-content-item.component.ts:16]: Thank you

**Pull request merged**